client/quick-links/style.scss

```diff
@@ -0,0 +1,70 @@
+.woocommerce-page .woocommerce-quick-links {
+	$background-color: $white;
+
+	background-color: $background-color;
+
+	.components-card__body.is-size-large {
+		padding: 0;
+
+		.woocommerce-list {
+			margin: 0;
+			padding: 0;
+
+			.woocommerce-list__item {
```

Comment:
These selectors are 4 and 5 levels deep, any way to make it less?

Reply:
I've improved this a little bit.